orc8r/gateway/python/magma/magmad/config_manager.py

```diff
@@ -109,19 +110,31 @@ def did_mconfig_change(serv_name):
             return mconfig.configs_by_key.get(serv_name) != \
                 self._mconfig.configs_by_key.get(serv_name)
 
-        # Reload magmad configs locally
-        if did_mconfig_change('magmad'):
+        def restart_magmad():
```

Comment:
Does this function restart magmad or only the dynamic servicecs loaded from the magmad config?